fr/conf.py

```diff
@@ -117,7 +117,7 @@
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-html_favicon = 'en/_static/favicon.ico'
+html_favicon = '_static/favicon.ico'
```

Comment:
No, it doesn't work. I've got a warning like this: "copying static files... WARNING: favicon file 'fr/_static/favicon.ico' does not exist". 
I use the command: D:\phalcon_docs\fr>make dirhtml
I'm using Sphinx 1.3b2 with Python 2.7 under Windows 10.
Maybe it's due to my tools.